boards/arm/cc3220sf_launchxl/doc/index.rst

```diff
@@ -271,6 +271,17 @@ example.
 See the document `Simplelink WiFi Certificates Handling`_ for details on
 using the TI UniFlash tool for certificate programming.
 
+Limitations
+***********
+The following features are not yet supported:
```

Comment:
better to say which version of Zephyr (in case we forget to update this later):
```suggestion
The following features are not supported in Zephyr v1.14:
```